TestApi/TestApi.UnitTests/Services/ServicesTestBase.cs

```diff
@@ -27,7 +27,8 @@ public abstract class ServicesTestBase
         protected Mock<ICommandHandler> CommandHandler;
         protected Mock<IConfiguration> Configuration;
         protected Mock<IOptions<UserGroupsConfiguration>> GroupsConfig;
-        protected Mock<ILogger<AllocationService>> Logger;
+        protected Mock<ILogger<AllocationService>> AllocationLogger;
```

Comment:
Am I reading this wrong, but does it look like you have more than one logger in a single class?

Reply:
its a base for various services